initfiles/componentfiles/configschema/xsd/hardware.xsd

```diff
@@ -1,92 +1,97 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema
-    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
-    xmlns:hpcc="someuri">
-  <xs:include schemaLocation="types.xsd"/>
-  <xs:complexType name="hardware" hpcc:class="component" hpcc:category="" hpcc:displayName="Hardware Configuration">
-    <xs:sequence>
-      <xs:element name="Hardware" hpcc:docid="????" hpcc:doc="Section to describe the hardware that composes the environment">
-        <xs:complexType>
-          <xs:sequence>
-            <xs:element name="ComputerType" maxOccurs="unbounded" hpcc:displayName="Defined computer types"  hpcc:class="valueSet">
-              <xs:complexType>
-                <xs:attribute name="name" type="xs:string" use="required"/>
-                <xs:attribute name="nicSpeed" type="xs:nonNegativeInteger"/>
-                <xs:attribute name="manufacturer" type="xs:string"/>
-                <xs:attribute name="memory" type="xs:nonNegativeInteger"/>
-                <xs:attribute name="computerType" type="xs:string"/>
-                <xs:attribute name="opSys">
-                  <xs:simpleType>
-                    <xs:restriction base="xs:string">
-                      <xs:enumeration value="linux"/>
-                      <xs:enumeration value="solaris"/>
-                      <xs:enumeration value="W2K"/>
-                    </xs:restriction>
-                  </xs:simpleType>
-                </xs:attribute>
-              </xs:complexType>
-            </xs:element>
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
```

Comment:
we need to come up w a better uri for the hpcc ns

Reply:
you did put one in, make sure we use the same placeholder throughout, and let's work on establishing a real one

Reply:
I meant if we added a real URL when I said if we put one in. I'd still like to address this issue across all XSDs. I opened HPCC-20214 to make sure it's not forgotten.

@rpastrana